infra/scoped/api-stage.tf

```diff
@@ -1,10 +1,28 @@
-resource "aws_api_gateway_deployment" "deployment" {
+resource "aws_api_gateway_deployment" "identity_deployment" {
   rest_api_id = aws_api_gateway_rest_api.identity.id
   description = timestamp()
 }
 
-resource "aws_api_gateway_stage" "v1" {
+resource "aws_api_gateway_stage" "identity_v1" {
   rest_api_id   = aws_api_gateway_rest_api.identity.id
-  deployment_id = aws_api_gateway_deployment.deployment.id
-  stage_name    = "v1"
+  deployment_id = aws_api_gateway_deployment.identity_deployment.id
+  stage_name    = local.identity_v1
+}
+
+resource "aws_api_gateway_domain_name" "identity_v1" {
+  certificate_arn = aws_acm_certificate_validation.identity_api_v1.certificate_arn
+  domain_name     = local.identity_v1_hostname
+
+  tags = merge(
+    local.common_tags,
+    map(
```

Comment:
You can use map literals in Terraform now BTW. e.g, `{name = local.identity_v1_hostname}`

Reply:
Good shout, effecting that change in https://github.com/wellcomecollection/identity/pull/6.